### PR TITLE
Add dedicated page for hwa issues

### PR DESCRIPTION
**Pull request by felix920506**
the list of known issues and limitations can make the page hard to read if it gets too long.
This PR moves those things into a dedicated page.

### CODE REVIEW

File: docs/general/administration/hardware-acceleration/amd.md [hunk 1]
```diff
@@ -464,7 +436,7 @@ Root permission is required.
          image: jellyfin/jellyfin
          user: 1000:1000
          group_add:
-           - "122" # Change this to match your "render" host group id and remove this comment
+           - '122' # Change this to match your "render" host group id and remove this comment
```
Comment (JPVenson) [inline docs/general/administration/hardware-acceleration/amd.md:439]: why this change?

Comment (felix920506) [reply docs/general/administration/hardware-acceleration/amd.md:439]: Vscode being stupid

Comment (JPVenson) [reply docs/general/administration/hardware-acceleration/amd.md:439]: in that case, revert pls

File: docs/general/administration/hardware-acceleration/intel.md [hunk 2]
```diff
@@ -644,19 +589,19 @@ The devices in Kubernetes are added as host path mounts, they are not separated
...
-               - name: "render-device"
-                 mountPath: "/dev/dri/renderD128"
+               - name: 'render-device'
+                 mountPath: '/dev/dri/renderD128'
          volumes:
-           - name: "render-device"
+           - name: 'render-device'
              hostPath:
-               path: "/dev/dri/renderD128"
+               path: '/dev/dri/renderD128'
```
Comment (nyanmisaka) [inline docs/general/administration/hardware-acceleration/intel.md:604]: vscode being stupid again

File: docs/general/administration/hardware-acceleration/known-issues.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+title: Known Issues
+---
+
+# Known Hardware Acceleration Issues and Limitations
+
+This page lists all known issues and limitations of hardware acceleration with Jellyfin.
+
+## AMD on Windows
+
+1. AMD Radeon RX 5000 and newer RDNA series integrated and discrete GPUs have an Windows graphics driver issue ranging from **Adrenalin 24.1.1 to 24.4.1**. You may experience **playback failure** and observe an **error code of `-60`** in the FFmpeg log when transcoding and playing videos. The driver **Adrenalin 24.5.1** and newer drivers fix this issue.
```
Comment (Gauvino) [inline docs/general/administration/hardware-acceleration/known-issues.md:12]:
```suggestion
1. AMD Radeon RX 5000 and newer RDNA series integrated and discrete GPUs have a Windows graphics driver issue ranging from **Adrenalin 24.1.1 to 24.4.1**. You may experience **playback failure** and observe an **error code of `-60`** in the FFmpeg log when transcoding and playing videos. The driver **Adrenalin 24.5.1** and newer drivers fix this issue.
```

File: docs/general/administration/hardware-acceleration/known-issues.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+   - [Fedora](https://www.phoronix.com/news/Fedora-Disable-Bad-VA-API)
+
+   - [OpenSUSE](https://www.webpronews.com/fedora-and-opensuse-disable-gpu-accelerated-video-over-patent-concerns/)
+
+   - [Manjaro](https://forum.manjaro.org/t/stable-update-2022-12-06-kernels-mesa-plasma-cinnamon-nvidia-libreoffice-pipewire-virtualbox/128453)
+
+   This prevents you from using the AMD VA-API transcoding **on the server system**.
+
+   You can use our Docker image instead or install third-party Mesa driver package.
```
Comment (Gauvino) [inline docs/general/administration/hardware-acceleration/known-issues.md:28]:
```suggestion
   You can use our Docker image instead or install the third-party Mesa driver package.
```

File: docs/general/administration/hardware-acceleration/known-issues.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+   Alternatively, rebuild the Mesa driver with these options added to restore the stripped hardware codecs:
+
+   ```shell
+   -D video-codecs=all
+   ```
+
+## Intel on Windows
+
+1. Intel 11th Gen and newer UHD, Xe and ARC series integrated and discrete GPUs have an Windows graphics driver issue ranging from **31.0.101.5186 / 31.0.101.5234 to 31.0.101.5534**. You may encounter a **green or blue screen but normal sound** when transcoding and playing HDR videos that **require tone-mapping**. The **31.0.101.5590** and newer drivers fix this issue.
```
Comment (Gauvino) [inline docs/general/administration/hardware-acceleration/known-issues.md:38]:
```suggestion
1. Intel 11th Gen and newer UHD, Xe and ARC series integrated and discrete GPUs have a Windows graphics driver issue ranging from **31.0.101.5186 / 31.0.101.5234 to 31.0.101.5534**. You may encounter a **green or blue screen but normal sound** when transcoding and playing HDR videos that **require tone-mapping**. The **31.0.101.5590** and newer drivers fix this issue.
```

File: docs/general/administration/hardware-acceleration/known-issues.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+## Intel on Linux
+
+1. Intel Gen 11 [**Jasper Lake**](https://ark.intel.com/content/www/us/en/ark/products/codename/128823/products-formerly-jasper-lake.html) and [**Elkhart Lake**](https://ark.intel.com/content/www/us/en/ark/products/codename/128825/products-formerly-elkhart-lake.html) platforms (e.g. N5095, N5105, N6005, J6412) have quirks when using video encoders on Linux. The [Low-Power Encoding](/docs/general/administration/hardware-acceleration/intel#low-power-encoding) mode MUST be configured and enabled for correct VBR and CBR bitrate control that is required by Jellyfin.
+
+   - Ticket: [https://gitlab.freedesktop.org/drm/intel/-/issues/8080](https://gitlab.freedesktop.org/drm/intel/-/issues/8080)
+
+2. The default kernel 5.15 that comes with Ubuntu 22.04 LTS has a regression on Intel Gen 11 graphics (ICL, JSL and EHL) that prevent you from using the Low-Power encoding mode. Linux 5.16+ is not affected.
```
Comment (Gauvino) [inline docs/general/administration/hardware-acceleration/known-issues.md:46]:
```suggestion
2. The default kernel 5.15 that comes with Ubuntu 22.04 LTS has a regression on Intel Gen 11 graphics (ICL, JSL and EHL) that prevents you from using the Low-Power encoding mode. Linux 5.16+ is not affected.
```

File: docs/general/administration/hardware-acceleration/known-issues.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+9. The kernel support for Intel Gen 12.7 MTL is incomplete before Linux 6.7.
+
+10. The LTS kernel range 6.6.26 - 6.6.32 and the stable kernel range 6.8.5 - 6.9.3 have i915 driver bugs, which may cause problems on Intel Gen 12.5 DG2 / ARC A-series GPUs. If you are affected, please upgrade to kernel 6.6.33+ (LTS) or 6.9.4+. Ubuntu 24.04 with kernel versions 6.8.0-38+ are also affected by this issue, please downgrade to 6.8.0-36 until a fix is released.
+
+    - Issue: [https://github.com/jellyfin/jellyfin/issues/11380](https://github.com/jellyfin/jellyfin/issues/11380)
+    - Ubuntu bug: [https://bugs.launchpad.net/ubuntu/+source/linux/+bug/2072755](https://bugs.launchpad.net/ubuntu/+source/linux/+bug/2072755)
+
+## Nvidia
+
+Consumer targeted [Geforce and some entry-level Quadro](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) cards have an artificial limit on the number of concurrent NVENC encoding sessions. This restriction can be circumvented by applying an [unofficial patch](https://github.com/keylase/nvidia-patch) to the NVIDIA Linux and Windows driver.
```
Comment (Gauvino) [inline docs/general/administration/hardware-acceleration/known-issues.md:85]:
```suggestion
Consumer targeted [Geforce and some entry-level Quadro](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) cards have an artificial limit on the number of concurrent NVENC encoding sessions. This restriction can be circumvented by applying a [unofficial patch](https://github.com/keylase/nvidia-patch) to the NVIDIA Linux and Windows driver.
```

**Review by Gauvino — CHANGES_REQUESTED** [review]
Some minor typos change I think

File: docs/general/administration/hardware-acceleration/known-issues.md (hunk 7 above)
Comment (felix920506) [reply docs/general/administration/hardware-acceleration/known-issues.md:85]: This one is how it's supposed to be, not a typo.

https://www.scribbr.com/commonly-confused-words/a-vs-an/

Comment (Gauvino) [reply docs/general/administration/hardware-acceleration/known-issues.md:85]: For this One I was not sure thanks.

**Review by Gauvino — APPROVED** [review]
LGTM !

File: docs/general/administration/hardware-acceleration/intel.md [hunk 8]
```diff
@@ -583,7 +528,7 @@ What you need to do is pass the host's `render` group id to Docker and modify th
          image: jellyfin/jellyfin
          user: 1000:1000
          group_add:
-           - "122" # Change this to match your "render" host group id and remove this comment
+           - '122' # Change this to match your "render" host group id and remove this comment
```
Comment (nyanmisaka) [inline docs/general/administration/hardware-acceleration/intel.md:531]: this

**Review by nyanmisaka — APPROVED** [review]

**Review by joshuaboniface — APPROVED** [review]

File: docs/general/administration/hardware-acceleration/known-issues.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+   - Ticket: [https://gitlab.freedesktop.org/drm/intel/-/issues/7627](https://gitlab.freedesktop.org/drm/intel/-/issues/7627)
+
+   - Fixed by: [drm/i915: improve the catch-all evict to handle lock contention](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=3f882f2d4f689627c1566c2c92087bc3ff734953)
+
+4. The current Debian 11 and Ubuntu 22.04 LTS may not have the required GPU firmware for Intel 12th Gen processors and ARC GPU. Ubuntu 22.04 LTS can be switched to use Ubuntu's Hardware Enablement Stack (see #2 above) which uses a newer kernel that supports Intel 12th Gen+ and ARC GPUs.
+
+5. The kernel support for Intel Gen 12 TGL graphics is incomplete before Linux 5.9.
+
+6. The kernel support for Intel Gen 12 DG1 is incomplete in upstream. Intel DKMS and custom iHD driver are required.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-acceleration/known-issues.md:70]: Is this still true? Seems a bit weird that Gen 12 DG1 still needs custom drivers while newer hardware is already fully supported?

**Review by Shadowghost — APPROVED** [review]

File: docs/general/administration/hardware-acceleration/known-issues.md (hunk 9 above)
Comment (felix920506) [reply docs/general/administration/hardware-acceleration/known-issues.md:70]: I don't know. I straight copied the text from the Intel HWA page. This can be updated in a later PR.

Comment (felix920506) [reply docs/general/administration/hardware-acceleration/known-issues.md:70]: It would be best for nyan to answer this question.
